bears/c_languages/ClangComplexityBear.py

```diff
@@ -1,9 +1,10 @@
 from clang.cindex import Index, CursorKind
 
 from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
 from coalib.results.Result import Result
 from coalib.results.SourceRange import SourceRange
-from bears.c_languages.ClangBear import clang_available, ClangBear
+from ClangBear.ClangBear import clang_available, ClangBear
```

Comment:
This is a wip, ill come back to clang bears once im  getting everything else done, ill open an issue and mark this as wip.